scripts/pyro_camera_server.py

```diff
@@ -5,11 +5,19 @@
 pyro_camera_local.yaml config file. The camera servers should be started after the name server,
 but before POCS.
 """
+import os, sys
 import argparse
-
 from huntsman.utils.pyro import run_camera_server
 
 if __name__ == "__main__":
+    
+    #Mount the SSHFS 
+    sys.path.append(
+            os.path.join(os.path.expandvars('$HUNTSMAN_POCS'), 'scripts'))
```

Comment:
This is unnecessary, I think? You're not executing any scripts, just calling the main function.

Reply:
/scripts is not installed as a python module, therefore I would not expect it to be on the default path.